docker-compose-services/chromedriver/README.md

```diff
@@ -0,0 +1,23 @@
+# Chrome Driver
+
+This recipe adds a Chrome Driver container to a project.
+
+This will allow you to have Drupal's Functional Javascript tests, among other things.
```

Comment:
We should use configuring Drupal's FunctionalJavascript test suite as an example, but in reality, I also used this for Behat with WebDriver support. The possibility is the sky's limit.